macros/edr/tests/on_run_end/insert_metrics.sql

```diff
@@ -0,0 +1,12 @@
+{% macro insert_metrics() %}
+  {% set metrics = elementary.get_cache("tables").get("metrics") %}
+  {% set database_name, schema_name = elementary.get_package_database_and_schema() %}
+  {%- set target_relation = adapter.get_relation(database=database_name, schema=schema_name, identifier='data_monitoring_metrics') -%}
```

Comment:
Those metrics [aren't being used in the tests](https://github.com/elementary-data/dbt-data-reliability/blob/3c4ce9e5d646ae9ac48fbf247a439d192c645038/macros/edr/data_monitoring/anomaly_detection/get_anomaly_scores_query.sql#L55) for now because they don't have metric properties.